recipes/list_items/contact_info/contact_info.vue

```diff
@@ -26,7 +26,10 @@
               :overlay-icon="avatar.icon"
               :overlay-text="avatar.text"
               overlay-class="d-mn4 d-ba d-baw4 d-bc-white d-box-unset"
-              :avatar-class="['d-ba d-baw4 d-bc-white d-bar-pill', { 'd-mln24': index > 0 }]"
+              :avatar-class="['d-baw4 d-bc-white d-bar-pill d-ba d-bc-transparent', {
+                'd-mln24': index > 0,
+                'd-bc-brand': !!avatar.halo,
```

Comment:
Please add halo into the avatarList description comment so it shows up in documentation.

Reply:
Im wondering what is avatar.halo? 
Using !! indicates me it is probably a object, what it contains? do we have some info about this?

Reply:
updated doc to include halo in the description. Using !! to handle in case it's undefined